sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Shared/WebPubSubStatusCode.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebPubSub.Common
+{
+    /// <summary>
+    /// The general enum to represent response status.
+    /// DO set to Success for backward capability.
+    /// Has mapping relationship to <see cref="WebPubSubErrorCode"/>.
+    /// </summary>
+    internal enum WebPubSubStatusCode
+    {
+        /// <summary>
+        /// Start from -1 to make values map <see cref="WebPubSubErrorCode"/>.
+        /// </summary>
+        Success = -1,
```

Comment:
@tg-msft For backward capability, I think we need to uniform the new enum code same as existing, Will this `Success` start from -1 a problem? We make this field internal in `WebPubSubEventResponse` and set to Success, so other classes won't accidently use `code` in future and leads to deserialization problem.

Reply:
The default value of a variable of this type will be `Unauthorized` (the member with value 0). Will that be an issue somewhere?

Reply:
Modified to start from 0 and make `Success` as default.